docs/userguides/legalhold.md

```diff
@@ -0,0 +1,96 @@
+ - To view a list of legal hold matters for your organization, including the matter ID, use the following command:
+   `code42 legal-hold list`
+ - To see a list of all the custodians currently associated with a legal hold matter, enter `code42 legal-hold show <matterID>`.
+
+
+## Get CSV template
+
+To add multiple custodians to a legal hold matter:
+
+1. Generate a CSV template. Below is an example command for generating a template to use to add custodians to legal hold matter. Once generated, the CSV file is saved to your current working directory.
+    `code42 legal-hold bulk generate-template add`
+
+    To generate a template to use to remove custodians from a legal hold matter, for example:
+
+    `code42 legal-hold bulk generate-template remove`
+
+    The CSV templates for adding and removing are the same.
+
+2. Use the CSV template to enter the matter ID(s) and Code42 usernames for the custodians you want to add to the matters.
+To get the ID for a matter, enter `code42 legal-hold list`.
+3. Save the CSV file.
+
+## Add custodians to a legal hold matter
+
+You can add one or more custodians to a legal hold matter using the Code42 CLI.
+
+### Add multiple custodians
+Once you have entered the matter ID and user information in the CSV file, use the bulk add-user command with the CSV file path to add multiple custodians at once. For example:
+
+`code42 legal-hold bulk add-user /Users/astrid.ludwig/add_users_to_legal_hold.csv`
+
+### Add a single custodian
+
+To add a single custodian to a legal hold matter, use the following command as an example:
+
+`code42 legal-hold add-user --matter-id 123456789123456789 --username tanya.rowe@example.com`
+
+#### Options
+
+ - `--matter-id` (required):   The identification number of the legal hold matter. To get the ID for a matter, run the command code42 legal-hold list.
+ - `--username` (required):    The Code42 username of the custodian to add to the matter.
+ - `--profile` (optional):     The profile to use to execute the command. If not specified, the default profile is used.
+
+## Release custodians
+You can [release one or more custodians](https://support.code42.com/Administrator/Cloud/Configuring/Create_a_legal_hold_matter#Release_or_reactivate_custodians) from a legal hold matter using the Code42 CLI.
+
+### Release multiple custodians
+
+To release multiple custodians at once:
+
+1. Enter the matter ID(s) and Code42 usernames to the [CSV file template you generated](#get-csv-template).
+2. Save the file to your current working directory.
+3. Use the `bulk remove-user` command with the file path of the CSV you created. For example:
+    `code42 legal-hold bulk remove-user /Users/hope.rodriguez/remove_users_from_legal_hold.csv`
+
+### Release a single custodian
+
+Use `remove-user` to remove a single custodian. For example:
+
+`code42 legal-hold remove-user --matter-id  123456789123456789 --username mercedes.simmons@example.com`
```

Comment:
Mercedes Simmons!!

imo, We should refrain from using real people's name in documentation, unless authorised.

Reply:
this is not a real person, these are names that are generated and appear in our demo environment.

Reply:
I decided to make the user info generic since I myself found the names a bit distracting (this copy was mostly all pulled from the support site doc that already existed).